tests/test_session.py

```diff
@@ -866,3 +879,122 @@ def token_transfer_method(req: BaseRequest, _: bool, __: Dict[str, Any]):
     assert len(response.headers.get("st-access-token", [])) == 0
     assert len(response.headers.get("st-refresh-token", [])) == 0
     assert len(response.headers.get("set-cookie", [])) > 0
+
+
+async def test_access_token_cookie_is_cleared_if_refresh_is_called_without_refresh_token(
+    driver_config_client: TestClient,
+):
+    init_args = get_st_init_args(
+        [
+            session.init(
+                anti_csrf="VIA_TOKEN",
+                get_token_transfer_method=lambda _, __, ___: "cookie",
+            )
+        ]
+    )
+    init(**init_args)
+    start_st()
+
+    response = driver_config_client.post("/auth/session/refresh")
+
+    assert response.status_code == 401
+    response_cookies = extract_all_cookies(response)
+    assert response_cookies["sAccessToken"]["value"] == ""
+    assert (
+        response_cookies["sAccessToken"]["expires"] == "Thu, 01 Jan 1970 00:00:00 GMT"
+    )
+
+
+async def test_access_and_refresh_tokens_are_cleared_if_multiple_tokens_are_passed_to_refresh_endpoint(
+    driver_config_client: TestClient,
+):
+
+    init_args = get_st_init_args(
+        [
+            session.init(
+                anti_csrf="VIA_TOKEN",
+                get_token_transfer_method=lambda _, __, ___: "cookie",
+                older_cookie_domain="example.com",
+            )
+        ]
+    )
+    init(**init_args)
+    start_st()
+
+    response = driver_config_client.post("/create")
+    cookies = extract_all_cookies(response)
+
+    assert "sAccessToken" in cookies
+    assert "sRefreshToken" in cookies
+
+    assert "anti-csrf" in response.headers
+    assert "front-token" in response.headers
+
+    cookiejar = cookiejar_from_dict({})  # type: ignore
+    cookiejar.set("sAccessToken", cookies["sAccessToken"]["value"])  # type: ignore
+    cookiejar.set("sRefreshToken", cookies["sRefreshToken"]["value"])  # type: ignore
```

Comment:
I don't think it changes anything, but this one should have the path if the other one also has it.

Reply:
Added it.